src/containers/board-container.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Board } from '../components/board.js';
+import {
+    getNewBoard,
+    flipTheLights,
+    getLights,
+    checkForWin,
+} from '../game-logic.js';
+
+export class BoardContainer extends React.Component {
+    state = {
+        numMoves: 0,
+        hasWon: false,
+    };
+
+    board = this.props.startingBoard ? this.props.startingBoard : getNewBoard();
+
+    shouldBeOn = (row, col) => {
+        return this.board[row][col];
+    };
+
+    flip = (row, col) => {
+        this.board[row][col] = !this.board[row][col];
+    };
```

Comment:
Just double-checking — are `shouldBeOn` and `flip` still being used? Couldn't find a reference to them.

Reply:
Nope, good catch!